cocotb/clock.py

```diff
@@ -59,12 +78,27 @@ def __init__(self, signal, period, units=None):
 
     @cocotb.coroutine
     def start(self, cycles=0):
+        """Clocking coroutine.  Start driving your clock by forking a call to
+        this.
+
+        Args:
+            cycles (int, optional): Cycle clock ``cycles`` number of times.
+                Defaults to 0, meaning will cycle forever.
```

Comment:
That's true, though this is a change that should go into another pull request.

Reply:
I will make the change as it actually shouldn't affect the behavior of it as `None` and 0 are both 'falsey', so it will just provide a difference in the documentation.